tests/main/base-policy/task.yaml

```
@@ -53,3 +59,11 @@ execute: |
     if os.query is-pc-amd64; then
         test-snapd-sh-other18.sh -c 'ls -l /var' 2>&1 | NOMATCH "Permission denied"
     fi
+
+    # core26 chose to ship uutils-coreutils at their native
+    # installation location used by cargo. the template was updated
+    # accordingly to allow them.
+    echo "Check if expected coreutils paths are allowed"
+    test-snapd-sh-core26.sh -c '/usr/bin/true' | NOMATCH "Permission denied"
+    # template doesn't contain permissions for whoami
+    test-snapd-sh-core26.sh -c '/usr/bin/whoami' 2>&1 | MATCH "Permission denied"
```

actually, looking at it again, we could be more thorough:

```suggestion
    # /usr/bin/true - usually a symlink to /usr/lib/cargo/bin/coreutils/true
    # /usr/lib/cargo/bin/coreutils/true - true from coreutils-uutils
    # /usr/bin/gnutrue - GNU coreutils
    for variant in /usr/bin/true /usr/bin/gnutrue /usr/lib/cargo/bin/coreutils/true; do
        test-snapd-sh-core26.sh -c "$variant"
    done
    
    # template doesn't contain permissions for whoami
    for variant in /usr/bin/whoami /usr/bin/gnuwhoami /usr/lib/cargo/bin/coreutils/whoami; do
        test-snapd-sh-core26.sh -c "$variant" 2>&1 | MATCH "Permission denied"
    done
```